src/components/community/comments/index.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FC, memo, useEffect, useState } from 'react';
+import { compose } from 'redux';
+import FdkLink from '@fellesdatakatalog/link';
+import {
+  useGetThreadByIdQuery,
+  useGetUserQuery,
+  usePostCommentMutation
+} from '../../../api/user-feedback-api/comments';
+import { withAuth, Props as AuthProps } from '../../../providers/auth';
+import SC from './styled';
+import Buttons from '../../buttons/styled';
+import Composer from './components/composer';
+import CommentCard from './components/comment-card';
+import { CommunityPost } from '../../../types';
+import translations from '../../../lib/localization';
+import env from '../../../env';
+
+const { FDK_COMMUNITY_BASE_URI } = env;
+
+interface ExternalProps {
+  entityId: string;
+}
+
+interface Props extends AuthProps, ExternalProps {}
+
+const EntityComments: FC<Props> = ({ entityId, authService }) => {
+  const [postComment] = usePostCommentMutation();
+  const [rootComments, setRootComments] = useState<CommunityPost[]>([]);
```

Comment:
En state for rootComments og replies burde være unødvendig, dette ligger jo allerede i redux-state.

Ved en update av thread i redux vil trigge en re-rendering av denne komponenten, deretter oppdaterer du local-state som trigger en ny re-render av komponenten.

Beste vil være å bruke selectors som gir rootComments og replies slik at dette ikke ligger i state i komponenten i tillegg.

Reply:
Såklart, takk for at du så.